### ISSUE TITLE
Feature Request: Forced list

### ISSUE THREAD

**Issue by qazarius** [jshackles/idle_master#207]
List of appids for idling in the first place

**Comment by dmlemos** [comment]
Yeah this would be great. I would prefer to idle the games I don't want to play, exhausting all the cards before moving to games I really want to play and review.

**Comment by AlexanderSharykin** [comment]
take a look at my fork of IM
https://github.com/AlexanderSharykin/CardIdleRemastered
Card Idle is designed to simplify selection of games to idle

**Comment by Tsunamical** [comment]
Yeah, I've been thinking about rewriting a few parts that are poorly written which is why I took all of my PRs down, and jshackles has pretty much abandoned new development on this.
